tests/core/atoms/test_skzcam.py

```diff
@@ -1,146 +1,555 @@
+        center_position,
+        adsorbate_vector_from_slab,
+    ) = get_cluster_info_from_slab(
+        FILE_DIR / "NO_MgO.poscar.gz", adsorbate_idx=[0, 1], slab_center_idx=[32, 33]
+    )
+
+    # Check adsorbate matches reference
+    np.testing.assert_allclose(
+        adsorbate.get_positions(),
+        np.array(
+            [
+                [5.39130495, 4.07523845, 15.96981134],
+                [5.88635842, 4.84892196, 16.72270959],
+            ]
+        ),
+        rtol=1e-05,
+        atol=1e-07,
+    )
+    assert np.all(adsorbate.get_atomic_numbers().tolist() == [7, 8])
+
+    # Check slab matches reference
+    np.testing.assert_allclose(
+        slab.get_positions()[::10],
+        np.array(
+            [
+                [0.0, 6.33073849, 7.5],
+                [0.0, 4.22049233, 9.61024616],
+                [4.2206809, 6.32743192, 11.73976183],
+                [4.2019821, 4.21892378, 13.89202884],
+                [0.0, 2.11024616, 9.61024616],
+                [4.22049233, 0.0, 7.5],
+                [4.22098271, 2.10239745, 13.86181098],
+            ]
+        ),
+        rtol=1e-05,
+        atol=1e-07,
+    )
+    assert np.all(
+        slab.get_atomic_numbers().tolist()
+        == [
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            12,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+            8,
+        ]
+    )
+
+    # Check first atom index of slab
+    assert slab_first_atom_idx == 30
+
+    # Check center_position matches reference
+    np.testing.assert_allclose(
+        center_position,
+        np.array([1.06307888, -1.06176564, 2.4591779]),
+        rtol=1e-05,
+        atol=1e-07,
+    )
+
+    # Check vector distance of adsorbate from first center atom (corresponding to first atom index) of slab matches reference
+    np.testing.assert_allclose(
+        adsorbate_vector_from_slab,
+        np.array([1.18932285, -0.14368533, 2.0777825]),
+        rtol=1e-05,
+        atol=1e-07,
+    )
+
+
+@pytest.mark.skipif(chemshell_module is None, reason="py-ChemShell is not installed")
```

Comment:
This works. That said, you may also be interested in knowing about `importorskip`: https://docs.pytest.org/en/stable/how-to/skipping.html#skipping-on-a-missing-import-dependency